src/settings_gui.cpp

```diff
@@ -486,6 +486,11 @@ struct GameOptionsWindow : Window {
 				break;
 			}
 
+			case WID_GO_BASE_MUSIC_OPEN_JUKEBOX: {
```

Comment:
*technically*, as you're not declaring any variables inside the case block, there's no need for `{}`. Don't mind either way though